////////////////////////////////////////////////////////////////////////////////////////////////////
// @breif C_WaitingRoomEnterHandler class
////////////////////////////////////////////////////////////////////////////////////////////////////


#pragma once
#include "Packet/Protocol.pb.h"


////////////////////////////////////////////////////////////////////////////////////////////////////
// @breif C_WaitingRoomEnterHandler class
////////////////////////////////////////////////////////////////////////////////////////////////////
class C_WaitingRoomEnterHandler
{
public:
	// HandlerRun
	static AtBool Handle( PacketSessionPtr& session, Protocol::C_WaitingRoomEnter& pkt );
};
